test/videojs-http-streaming.test.js

```diff
@@ -3715,6 +3715,47 @@ QUnit.test('seekToProgramTime will seek to time if buffered', function(assert) {
   this.clock.tick(2);
 });
 
+QUnit.test('have a default syncpoint if switching renditions before one is selected', function(assert) {
+  assert.timeout(3000);
+  const done = assert.async();
+
+  this.player.muted(true);
+  this.player.src({
+    src: 'live.m3u8',
+    type: 'application/vnd.apple.mpegurl'
+  });
+  this.clock.tick(1);
+  this.player.play();
```

Comment:
calling play or using autoplay causes us to be in a has played state by the time we switch the playlist, meaning the playlist won't get a syncinfo.